client/src/containers/HomePage.tsx

```diff
@@ -127,10 +127,10 @@ const HomePage: React.FC<HomePageProps> = ({ useNewPortfolioMethod }) => {
   const history = useHistory();
 
   const labelText = (
-    <Trans>Enter an NYC address and find other buildings your landlord might own:</Trans>
+    <Trans>Enter an NYC address and find other buildings your landlord might own in NYC:</Trans>
```

Comment:
```suggestion
    <Trans>Enter an address and find other buildings your landlord might own in NYC:</Trans>
```

or leave it as it was previously? the double NYC mention is a bit odd

Reply:
Oh yeah great point, that's much better!